packages/networkmanager.rb

```diff
@@ -28,7 +28,8 @@ class Networkmanager < Meson
   depends_on 'elogind'
```

Comment:
Update to  [1.44.4](https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/tags/1.44.4)?